_posts/2018-03-01-core-team-announcement.md

```diff
@@ -11,18 +11,18 @@ The year 2018 is well underway, with many exciting new technologies on the horiz
 The role of the Core Team can often be a confusing topic, especially with the various leadership structures employed by other cryptocurrencies. As the technology for a completely trustless society is not yet a reality, all projects, even decentralized ones, benefit from having individuals that can carry out tasks that require trust. A quick example of this is owning a domain name (like getmonero.org) or maintaining servers (on which getmonero.org is hosted). Ultimately, there is not yet a way for a domain name to be owned by a decentralized community, so someone has to be the name holding this domain. It's in scenarios like this where the Core Team are able to work on bridging the decentralized world we are trying to bring about with the centralized reality in which we are currently living.
 
 The responsibilities of the Core Team are as follows (in no particular order):
-    - Act as primary trusted arbiters of the Forum Funding System on behalf of the community, so as to ensure the completion of all projects to the satisfaction of the community.
-    - Manage the codebase of the Monero Project, which includes merging code on Github, keeping backups, and ensuring the safety, security, and free access of the code from any party.
-    - Steward the general donation fund, and spending the Monero there on anything they see fit to further the Monero Project.
-    - Act as trusted signers and distributors of reference clients for the Monero coin, and other related technologies.
-    - Set a direction and vision for the Monero Project
+- Act as primary trusted arbiters of the Forum Funding System on behalf of the community, so as to ensure the completion of all projects to the satisfaction of the community.

```

Comment:
How is this related in any way and went through code review?
@luigi1111 @rehrar

Reply:
What's the issue?

Reply:
Leading whitespaces were removed, should have been done in a different PR.